rest/src/main/kotlin/json/request/InteractionsRequests.kt

```diff
@@ -54,6 +54,12 @@ public data class AutoCompleteResponseCreateRequest<T>(
     val data: DiscordAutoComplete<T>
 )
 
+@Serializable
+public data class ModalResponseCreateRequest(
+    val type: InteractionResponseType,
```

Comment:
```suggestion
    val type: InteractionResponseType.Modal,
```
this could also be applied to other `ResponseCreateRequests`

Reply:
probably just take it out and put it inside the class body

Reply:
I think serialization would break that way. A default value would be a good option though.